packages/editor-ui/src/components/RunDataJson.test.ts

```diff
@@ -0,0 +1,61 @@
+import Vue from "vue";
+import { render, screen } from '@testing-library/vue';
+import RunDataJson from '@/components/RunDataJson.vue';
+
+// TODO: Investigate why this is needed
```

Comment:
is not this expected since you are fully rendering? have you tried a shallow render instead?

Reply:
In this case, that's what makes sense (rendering `vue-json-pretty`) so I can see what's the real output.
Apart from this, I don't like `shallowMount` because it's cheating and sometimes can trick you.
In **Vue Testing Library** there is no such thing anyways, you have the `render` method which is basically the `mount` from **Vue Test Utils**, if you want to leave out a child component from rendering you have to explicitly mock it (`shallowMount` would do automatically for all children which also strange)

What I could do to make it nicer is that moved this import to the test setup file so no one has to worry about it again when testing a component that uses `vue-json-pretty`